guide_drafts/trait_derives.md

```diff
@@ -226,106 +320,32 @@ fn main() {
 If we try to insert records without deriving `Insertable`, we would get the following error.
 
 ```rust
-
-error[E0277]: the trait bound `&diesel_demo::models::NewUser<'_>: diesel::query_builder::insert_statement::IntoInsertStatement<_, diesel::query_builder::insert_statement::Insert>` is not satisfied
-  --> src/bin/main.rs:31:30
+error[E0277]: the trait bound `&diesel_demo::models::NewUser<'_>: diesel::Insertable<diesel_demo::schema::users::table>` is not satisfied
+  --> src/bin/main.rs:29:10
    |
-31 |     diesel::insert(&new_user).into(users::table)
-   |                               ^^^^ the trait `diesel::query_builder::insert_statement::IntoInsertStatement<_, diesel::query_builder::insert_statement::Insert>` is not implemented for `&diesel_demo_cli::models::NewUser<'_>`
-```
-
-In the above error, the compiler is telling us there is an unsatisfied trait bound from `IntoInsertStatement`.
-`diesel::insert()` returns an `DeprecatedIncompleteInsertStatement`,
-which implements `into()`.
-`into()` specifies the table the `Insertable` data should be passed to.
-If your struct isn't `Insertable`, Rust will get confused and ask you implement this trait.
-
-Now we will look at how `Insertable` would be implemented by hand.
-This code was generated by the `cargo-expand` crate and slightly modified to reduce namespacing and increase readability.
-Required imports are not included for brevity.
-
-```rust
-#[derive(Queryable)]
-pub struct User {
-    pub id: i32,
-    pub first_name: String,
-    pub last_name: String,
-    pub email: String,
-}
-
-pub struct NewUser<'a> {
-    pub first_name: &'a str,
-    pub last_name: &'a str,
-    pub email: &'a str,
- }
-
-impl <'a, 'insert, DB> Insertable<users::table, DB> for &'insert NewUser<'a>
```

Comment:
I think that we should not remove the generated code examples, because they show how one could implement this traits manually. Maybe we should move this to a special section in the end of this guide or even to a additional guide?

Reply:
Whoops! Good catch. I thought I removed all of the generated impls. I agree in moving them to a separate guide.